src/localisation/localisation.c

```diff
@@ -1295,3 +1295,75 @@ int win1252_to_utf8(utf8string dst, const char *src, size_t srcLength, size_t ma
 
 	return result;
 }
+
+money32 string_to_money(char * string_to_monetise)
+{
+	const char* decimal_char = language_get_string(STR_LOCALE_DECIMAL_POINT);
+	char * text_ptr = string_to_monetise;
+	int i, j, sign;
+	//Remove everything except numbers decimal, and minus sign(s)
+	for (i = 0; text_ptr[i] != '\0'; ++i) {
+		while (!(
+			(text_ptr[i] >= '0' && text_ptr[i] <= '9') ||
+			(text_ptr[i] == decimal_char[0]) ||
+			(text_ptr[i] == '-') || 
+			(text_ptr[i] == '\0')
+		)) {
+			//move everything over to the left by one
+			for (j = i; text_ptr[j] != '\0'; ++j) {
+				text_ptr[j] = text_ptr[j + 1];
+			}
+			text_ptr[j] = '\0';
```

Comment:
I used it because https://github.com/OpenRCT2/OpenRCT2/pull/4166#issuecomment-245886180 and the few links below use it, so i'm sure there's a reason so many people have it. I'd prefer to keep it just to be safe.

http://stackoverflow.com/a/28609778/6569887
http://stackoverflow.com/a/35494834/6569887
http://stackoverflow.com/a/37278693/6569887
http://stackoverflow.com/a/8733511/6569887